js-quantities/index.d.ts

```diff
@@ -70,4 +70,4 @@ declare namespace Qty {
     type UnitSource = Qty | string;
 }
 
-export default Qty;
+export = Qty;
```

Comment:
I think it's better here  to declare a module and then to update the tests according to a changed API.
```ts
declare module 'js-quantities' {
    export interface QtyInstance extends Qty.Qty {}
    export const Qty: Qty.Type;
    export default Qty;
}
```
And use it like this
```ts
import * as quantities from "js-quantities";
let Qty = quantities.Qty;
let qtyCopy: quantities.QtyInstance;

```